recipes/openssl/3.x.x/conanfile.py

```diff
@@ -85,11 +85,13 @@ class OpenSSLConan(ConanFile):
         "no_whirlpool": [True, False],
         "no_zlib": [True, False],
         "openssldir": [None, "ANY"],
+        "securitylevel": [None, "ANY"],
```

Comment:
Possible values are 0, 1, 2, 3, 4, 5. Default value is 1 in OpenSSL. Why not just provide this range instead of ANY?

https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_security_level.html

Reply:
Thanks for this suggestion.
I have pushed a small modification using a range, which also improves the input validation.